packages/apps/job-launcher/server/src/modules/auth/strategy/jwt.http.ts

```diff
@@ -26,28 +23,43 @@ export class JwtHttpStrategy extends PassportStrategy(Strategy, 'jwt-http') {
         ConfigNames.JWT_SECRET,
         'secretkey',
       ),
+      passReqToCallback: true,
     });
   }
 
-  public async validate(payload: {
-    refreshToken: string;
-    email: string;
-  }): Promise<UserEntity> {
-    const refreshToken = payload.refreshToken.toLowerCase();
-    const authEntity = await this.authService.getByRefreshToken(refreshToken);
+  public async validate(
+    @Req() request: any,
+    payload: { email: string; userId: number },
+  ): Promise<UserEntity> {
+    const auth = await this.authRepository.findOne(
+      {
+        userId: payload.userId,
+      },
+      {
+        relations: ['user'],
+      },
+    );
 
-    if (!authEntity?.user) {
-      throw new NotFoundException('User not found');
+    if (!auth?.user) {
+      throw new UnauthorizedException('User not found');
     }
 
-    if (authEntity?.user.status !== UserStatus.ACTIVE) {
+    if (auth?.user.status !== UserStatus.ACTIVE) {
       throw new UnauthorizedException('User not active');
     }
 
-    if (authEntity.status !== AuthStatus.ACTIVE) {
-      throw new UnauthorizedException('Token expired');
+    //check that the jwt exists in the database
+    const jwt = (request.headers['authorization'] as string).substring(7);
```

Comment:
Let's make sure that first 7 characters in header are `Bearer `.

Reply:
@Dzeranov fixed, please check